tests/integration/security/rbac/v2/basic/testdata/istio-extended-rbac-v2-rules.yaml.tmpl

```diff
@@ -23,29 +23,26 @@ spec:
 apiVersion: "rbac.istio.io/v1alpha1"
 kind: ServiceRole
 metadata:
-  name: access-a-http
+  name: access-a
```

Comment:
Rename it to "access-http"?

Reply:
I kinda want to use the same name for both service role to demonstrate the difference when referring to them is the `/`. Let me know if you still want to change them.